modules/virt-about-pci-passthrough.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
```

Comment:
Missing a directory; should be `* virt/virtual_machines/advanced_vm_management/configuring-pci-passthrough.adoc`

Reply:
Updated the content.